core/src/main/kotlin/org/evomaster/core/problem/rest/RestIndividual.kt

```diff
@@ -7,20 +7,45 @@ import org.evomaster.core.search.Action
 import org.evomaster.core.search.Individual
 import org.evomaster.core.search.gene.*
 import org.evomaster.core.search.service.Randomness
+import org.evomaster.core.search.service.tracer.TraceableElement
+import org.evomaster.core.search.service.tracer.TrackOperator
+
+
+class RestIndividual : Individual {
+
+    val actions: MutableList<RestAction>
+    val sampleType: SampleType
+    val dbInitialization: MutableList<DbAction>
+    val usedObjects: UsedObjects
+
+    constructor(
```

Comment:
why using a "constructor"? you know that you can have default parameters in the inputs?

Reply:
modified. thanks! but somehow, the class cannot be created with default values in java. So I modified a bit in a test "DbDirectIntWithSqlEMTest" of e2e-test as below         

RestIndividual withSQL = new RestIndividual(ind.getActions(), ind.getSampleType(), insertions, ind.getUsedObjects(), null, null);